gffquant/profilers/panda_profiler.py

```
@@ -0,0 +1,370 @@
+                    index=False,
+                    float_format="%.5f"
+                )  # noqa
+
+    def dump(
+        self,
+        out_prefix,
+        restrict_reports=None,
+        report_category=True,
+        report_unannotated=False,
+    ):
+        publish_reports = [
```

Ruff (0.8.0)

Local variable `publish_reports` is assigned to but never used

Remove assignment to unused variable `publish_reports`

(F841)